www/w3m/Portfile

```diff
@@ -49,12 +49,13 @@ if {${name} eq ${subport}} {
 subport w3m-devel {
     PortGroup       github 1.0
 
-    github.setup    tats w3m 2b59b9eb0af758d6c7f4886933fb494699d28dc6
-    version         20211027
+    github.setup    tats w3m c515ea8a47b62408943390f579b2f377464f6658
+    version         0.5.3+git20220429
```

Comment:
`+` is [an illegal character](https://trac.macports.org/ticket/65073) in `version`.

Reply:
If you change the version format in this manner, you must increase `epoch` to counteract the usual reasoning of `vercmp`.
```
$ ~/macports/macports-user-ryandesign/scripts/vercmp 20211027 0.5.3+git20220429
MacPorts considers 20211027 to be greater than 0.5.3+git20220429.
```

Reply:
Hi @ryandesign,

I have changed the version to use only the date portion of the tag on w3m's github project page.  Hopefully this fixes the issues.

Thanks,

-ranga